src/platform/windows.rs

```diff
@@ -117,6 +117,12 @@ pub trait WindowBuilderExtWindows {
 
     /// This sets `WS_EX_NOREDIRECTIONBITMAP`.
     fn with_no_redirection_bitmap(self, flag: bool) -> WindowBuilder;
+
+    /// Enables or disables drag and drop support (enabled by default). Will interfere with other crates
+    /// that use multi-threaded COM API (`CoInitializeEx` with `COINIT_MULTITHREADED` instead of
+    /// `COINIT_APARTMENTTHREADED`) on the same thread.
+    /// See https://docs.microsoft.com/en-us/windows/win32/api/objbase/nf-objbase-coinitialize#remarks for more information.
```

Comment:
One more note that's occurred to me since the initial review, and that should make it into the documentation:

Even with `with_drag_and_drop` set to `false`, Winit can still call `CoInitializeEx(COINIT_APARTMENTTHREADED)`. Right now that only happens if the user fullscreens a window (see the [`com_initialized`](https://github.com/rust-windowing/winit/blob/878c179761cbc536ad24906486430ed2bb1d6092/src/platform_impl/windows/window.rs#L833-L835) function and the functions that call it), but I can't make any guarantees about whether or not it'll be called elsewhere in Winit's codebase in the future. If the user needs to initialize COM with `COINIT_MULTITHREADED` they should do it before calling any Winit functions, just to be safe.